parts/linux/cloud-init/artifacts/aks-rsyslog

```diff
@@ -0,0 +1,40 @@
+/var/log/syslog
+/var/log/messages
+/var/log/kern.log
+{
+  rotate 5
+  daily
+  size 300M
```

Comment:
was this default? or just to keep a bit more? fine either way, just checking

Reply:
weekly -> daily, rotate 4 -> rotate 5, added size 300M